pages/404.js

```diff
@@ -0,0 +1,53 @@
+import { ErrorPage } from '@dts-stn/service-canada-design-system'
+
+export default function Custom404(props) {
+  return (
+    <>
+      <ErrorPage
+        lang={props.locale}
+        errType="404"
+        isAuth={!props.isAuth}
+        homePageLink={
+          props.locale === 'en' ? '/en/my-dashboard' : '/fr/mon-tableau-de-bord'
+        }
+        accountPageLink="/"
+      />
+    </>
+  )
+}
+
+export async function getStaticProps({ locale }) {
+  const langToggleLink = locale === 'en' ? '/fr/404' : '/404'
+
+  /* Place-holder Meta Data Props */
+  const meta = {
+    data_en: {
+      title: 'My Service Canada Account - 404',
```

Comment:
as per [AB#107587](https://dev.azure.com/VP-BD/DECD/_workitems/edit/107587), this should be `404 - My Service Canada Account, same thing for FR and the other error page

Reply:
Should be good now.